### ISSUE TITLE
Visual Keyboard Hint Missing when switching from trackpad to keyboard nav

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/balloons-and-static-electricity#536]
**Test device**
Mac11 (m1 chip)

**Operating System**
11.5.2

**Browser**
safari

**Problem description**
https://github.com/phetsims/qa/issues/706

After moving the balloons using the mouse, if you then switch to keyboard nav, the "press space bar to grab or release" hint will appear. However, once you press the space bar, the hint displaying which keys to use to move the balloon does not appear. 

**Steps to reproduce**
1. Using the mouse, move the yellow balloon onto the sweater. 
2. Then press the tab key. The pink highlight and "press space bar to grab or release" hint will appear.
3. Press space. The arrows/WASD visual keyboard hint will not appear on the screen.

**Visuals**
In the attached video, I used the two balloons mode so that you can see the difference in hints between a moved and unmoved balloon.

https://user-images.githubusercontent.com/87318828/133095037-7183d119-2f35-47f8-911b-61c5be06d3f0.mov

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.1/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-rc.1 2021-09-07 23:14:39 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Safari/605.1.15
Language: en-us
Window: 1199x683
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

**Comment by jbphet** [comment]
I believe that this is by design.  Once the balloon has moved, that hint is no longer shown, regardless of how it is moved.

Assigning to @jessegreenberg for potential confirmation.  @jessegreenberg - If the described behavior is indeed what we want, feel free to close.

**Comment by jbphet** [comment]
We reviewed this in today's sound design meeting, and @emily-phet confirmed that this is indeed the intended behavior.  Closing.
